index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="zxx">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <nav id="navbar">
```

Comment:
Please use the <header> semantic tag here instead. Because the content inside it contains non-navigation links. such as the <JS Documentation> logo.

Reply:
Hi, @gtekle from the cade I can see a semantic `<nav>` tag, you need to be more descriptive.